pcsx2/Recording/PadData.cpp

```diff
@@ -76,10 +76,6 @@ void PadData::SetNormalButtons(int port, std::vector<int> buttons)
 
 void PadData::SetNormalButton(int port, PadData_NormalButton button, int fpushed)
 {
-	if (port < 0 || 1 < port)
-	{
-		return;
-	}
```

Comment:
Once again, this looks like content that should be left alone until multi-tap support is added.  Though I'm not too concerns about this file since I basically completely remove it with the virtualpad overhaul.  So feel free to ignore this one!